### PR TITLE
[59]:.. Bindings for reuseaddr

### PR DESCRIPTION
**Pull request by Darrellbor**
PR for #59 
Implemented:
- [x] binding for `roc_receiver_set_reuseaddr`
- [x] binding for `roc_sender_set_reuseaddr`
- [x] test for `sender.SetReuseaddr()`
- [x] test for `receiver.SetReuseaddr()`

### CODE REVIEW

File: roc/sender_receiver_test.go [hunk 1]
```diff
@@ -92,6 +92,16 @@ func newTestEnv(t *testing.T) *testEnv {
 	return &e
 }
 
+func TestSenderSetReuseaddr(t *testing.T) {
```
Comment (ortex) [inline roc/sender_receiver_test.go:95]: I think it's better to move this test to sender_test

**Review by ortex — CHANGES_REQUESTED** [review]

**Comment by Darrellbor** [comment]
pushed the fixes, although the tests stopped, can you restart it

**Review by gavv — COMMENTED** [review]
Thanks for PR! A few comments from me too

**Comment by gavv** [comment]
BTW when request is ready for (re-)review, please leave a comment or press "request review" button to indicate that - otherwise it's hard to guess whether it's already time to look at the pushed changes.

**Comment by Darrellbor** [comment]
Noted @gavv

**Comment by gavv** [comment]
Thanks for update!

+1 from me.

@ortex

**Review by gavv — APPROVED** [review]

**Review by ortex — APPROVED** [review]

**Comment by ortex** [comment]
LGTM

File: roc/receiver_test.go [hunk 2]
```diff
@@ -53,3 +54,74 @@ func TestReceiver_Open(t *testing.T) {
...
+	cases := []struct {
+		name                 string
+		slot                 Slot
+		iface                Interface
+		receiverClosedBefore bool
+		enabled              bool
+		wantErr              error
+	}{
+		{
+			name:                 "Ok",
```
[the remark this answers is not in the document]
Comment (Darrellbor) [reply roc/receiver_test.go]: test-case name fixed

**Comment by gavv** [comment]
Thank you!

**Pull request merged**